spec/mcode.spec.ts

```diff
@@ -2464,6 +2464,59 @@ describe('checkHistologyMorphologyFilterLogic-dcis', () => {
     expect(extractedMCODE.getHistologyMorphologyValue()).toBe('dcis');
   });
 });
+describe('checkSecondaryCancerConditionLogic', () => {
```

Comment:
Not sure how necessary this is but previously we would have a test for each of the value results to make sure we actually included them all.

Reply:
That is a good point Robi. I do think since the logic for this functionality is fairly straightforward (not a bunch of ANDs and ORs), that it is okay to just make sure all of the lines and branches are covered.